examples/react-tldraw/.env

```diff
@@ -0,0 +1,5 @@
+VITE_YORKIE_API_ADDR='http://localhost:8080'
+VITE_YORKIE_API_KEY=''
+
+VITE_AWS_PRESIGNED_URL_ADDR=''
+VITE_AWS_DELETE_OBJECT_ADDR=''
```

Comment:
This example appears to have an AWS dependency. If it is possible, to keep the focus on integrating Tldraw and Yorkie, how about removing the non-essential dependency for the sake of the example?

Reply:
Thank you for your advice!

I removed `useMultiplayerAssets.ts` which is image/video storage API using AWS S3. And also removed related environment values too.